pkg/protocols/network/network.go

```diff
@@ -169,6 +175,23 @@ func (request *Request) Compile(options *protocols.ExecutorOptions) error {
 		}
 	}
 
+	// parse ports and validate
+	if request.Port != "" {
+		for _, port := range strings.Split(request.Port, ",") {
+			if port == "" {
+				continue
+			}
+			portInt, err := strconv.Atoi(port)
+			if err != nil {
+				return errorutil.NewWithErr(err).Msgf("could not parse port %v from '%s'", port, request.Port)
```

Comment:
Alternatively, we can proceed with the ones we can and skip the ones we can't parse. What do you think?

Reply:
earlier i did that . but since `compile()` is called when validating templates / loading templates . by erroring out we user will be able to detect such logical errors while creating nuclei template